modules/http/src/test/java/org/mule/module/http/functional/proxy/HttpProxyParamsTestCase.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.http.functional.proxy;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.mule.api.MessagingException;
+import org.mule.api.MuleEvent;
+import org.mule.module.http.functional.requester.AbstractHttpRequestTestCase;
+import org.mule.tck.junit4.rule.DynamicPort;
+
+import java.io.IOException;
+
+import static javax.servlet.http.HttpServletResponse.SC_OK;
+import static junit.framework.TestCase.fail;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.mule.module.http.api.HttpConstants.ResponseProperties.HTTP_STATUS_PROPERTY;
+
+// TODO - MULE-9563 - improve this suite starting a proxy-server and checking that the request went through
+public class HttpProxyParamsTestCase extends AbstractHttpRequestTestCase {
+
+    @Rule
+    public DynamicPort proxyPort = new DynamicPort("proxyPort");
+
+    @Override
+    protected String getConfigFile() {
+        return "http-request-proxy-config.xml";
```

Comment:
Create another config file exclusive to this feature, otherwise you're initialising a lot of unnecessary things.